sycl/unittests/CMakeLists.txt

```diff
@@ -46,6 +46,6 @@ add_subdirectory(Extensions)
 add_subdirectory(windows)
 add_subdirectory(event)
 add_subdirectory(buffer)
-add_subdirectory(context)
+add_subdirectory(context_device)
```

Comment:
Looks like `context_device` directory is missing in this PR.

Reply:
Fixed.